src/Components/01.Homepage/AboutUs.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+interface Props {
+}
+
+const AboutUs: React.FC<Props> = () => {
+  const header = 'What is Pinch?';
+
+  return (
+    <div className="AboutUs">
```

Comment:
typically CSS classes follow this-case instead of PascalCase so can you fix this classname please?